client/layout/masterbar/logged-in.jsx

```diff
@@ -96,6 +96,7 @@ export default React.createClass( {
 					{ this.translate( 'New Post' ) }
 				</Publish>
 				<Item
+					tipTarget="me"
```

Comment:
> Tip target seems a bit obscure to me, but maybe it's alright. @ockham do you have any thoughts as you were looking at react alternatives for component-tip?

Late to the party, sorry. Haven't looked deeply into [`React-tooltip`](https://github.com/wwayne/react-tooltip) yet, TBH. Reference (for others interested): #4072.